server/config/parameters.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2024 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"gopkg.in/src-d/go-errors.v1"
+
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/variables"
+)
+
+// init initializes or appends to SystemVariables as it functions as a global variable.
+// TODO: get rid of me, use an integration point to define new sysvars
+func init() {
+	InitConfigParameters()
+}
+
+// InitConfigParameters resets the postgresConfigVariables singleton in the sql package.
+// Currently, we append all of postgres configuration parameters to sql.SystemVariables.
+// This means that all of mysql system variables and postgres config parameters will be stored together.
+// TODO: issue with this approach is that there are two postgres parameters
+//
+//	have the same name as mysql variables that will override the mysql variables.
+func InitConfigParameters() {
```

Comment:
We could remove this function and merge it with `init`, since it looks like it's only called from `init`.